src/views/QuestionView.vue

```diff
@@ -242,16 +238,33 @@ export default {
             }
           });
           if (
-            result.data.questions.length < count &&
+            result.data.count < count &&
             (clean || !this.questionBuffer.includes(NO_QUESTION_LEFT))
           ) {
             dataToAdd.push(NO_QUESTION_LEFT);
           }
```

Comment:
I have a doubt about this condition. `result.data.count` is the total number of questions and count is the number of questions asked per request

Maybe it is worth doing a renaming. For example `count` could be renamed `pageSize`

Reply:
My bad, you put the correct end condition after. This lines can be deleted 👍